models/cart/src/index.ts

```diff
@@ -5,7 +5,7 @@ export * from './line-item/types';
 
 // Export models
 export * as Cart from './cart';
-export * as CartDraft from './cart/cart-draft';
+export * from './cart/cart-draft';
```

Comment:
I changed this to maintain the same imports for consumers. since the cart package exports main different things and has an extra `index.ts` file, I needed to change this export to maintain the same behavior for consumers

Reply:
I'm not sure I follow. 
Would it be ok to have a quick sync about this? 🙏

Reply:
sure I'm available for a sync anytime that is good for you